### PR TITLE
fix: replace colors with chalk

### LINKED ISSUE #948: unreadable output while using electron-rebuild
I am using the electron 13.0.0 and node.js v14.8.2
the terminal is zsh, font is SauceCodePro Nerd.
image

### PR DESCRIPTION
**Pull request by malept**
Fixes #948.

### CODE REVIEW

File: test/fixture/native-app1/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "ffi-napi": "2.4.5"
   },
   "dependencies": {
-    "@newrelic/native-metrics": "5.3.0",
+    "@newrelic/native-metrics": "7.0.2",
```
Comment (erickzhao) [inline test/fixture/native-app1/package.json:21]: is this related to the `colors` change, or related to the current CI windows failures?

File: yarn.lock [hunk 2]
```diff
@@ -1815,7 +1815,7 @@ colors@1.0.3:
   resolved "https://registry.yarnpkg.com/colors/-/colors-1.0.3.tgz#0433f44d809680fdeb60ed260f1b0c262e82a40b"
   integrity sha1-BDP0TYCWgP3rYO0mDxsMJi6CpAs=
 
-colors@^1.1.2, colors@^1.3.3:
+colors@^1.1.2:
```
Comment (erickzhao) [inline yarn.lock:1818]: is it concerning that we still have `colors` in the lockfile? is this from an upstream dep?

**Review by erickzhao — APPROVED** [review]
changes LGTM, looks like CI is still failing 🤔

File: test/fixture/native-app1/package.json (hunk 1 above)
Comment (malept) [reply test/fixture/native-app1/package.json:21]: I was trying to fix the CI Windows failures :disappointed:

File: yarn.lock (hunk 2 above)
Comment (malept) [reply yarn.lock:1818]: This is a transitive dependency from `semantic-release` :arrow_right: `marked_terminal` :arrow_right: `cli-table`

**Pull request merged**